### DISCUSSION TITLE
Pulling speed and altitude

### DISCUSSION THREAD

**Discussion by chanbennet** [Knio/pynmea2#134] [Q&A]
What are the attributes to parse out speed from GPRMC? 

```
 newdata[0:6] == "$GPRMC":
            newmsg=pynmea2.parse(newdata)
            lat=newmsg.latitude
            lng=newmsg.longitude
            spd=newmsg.spd_over_grnd_kmph
            gps_data['latitude'] = lat
            gps_data['longitude'] = lng
            gps_data['speed'] = spd
            gps = "Latitude=" + str(lat) + "and Longitude=" + str(lng) + "speed" + str(spd)
            print(gps)
            
```

Returns 
    spd=newmsg.spd_over_grnd_kmph
  File "/home/pi/.local/lib/python2.7/site-packages/pynmea2/nmea.py", line 155, in __getattr__
    raise AttributeError(name)
AttributeError: spd_over_grnd_kmph

**Comment by Knio** [comment]
These fields are defined here: https://github.com/Knio/pynmea2/blob/master/pynmea2/types/talker.py#L357
